q2_diversity/tests/test_alpha.py

```diff
@@ -439,6 +439,22 @@ def test_alpha_group_significance_forward_slash_in_metadata_col(self):
                                   'kruskal-wallis-pairwise-a%2Fb.csv')
             self.assertTrue(os.path.exists(csv_fp))
 
+    def test_sample_ids_in_error_message(self):
+        alpha_div = pd.Series([2.0, 4.0, 6.0], name='alpha-div',
+                              index=['sample1', 'sample2', 'sample3'])
+        md = qiime2.Metadata(
+            pd.DataFrame(
+                {'col1': [1, 2, 1],
+                 'col2': [4.2, 4.2, 4.3]},
+                index=pd.Index(['sample1', 'sample2', 'sample3'], name='id')))
+
+        with self.assertRaises(ValueError) as cm:
```

Comment:
Mainly for my own edification, but what was your reasoning behind choosing `cm` as the variable name here?

Reply:
hey @hagenjp i went ahead and updated this variable name to `ve` (which is commonly what we do for ValueError abbreviations)